packages/ckeditor5-engine/tests/controller/datacontroller.js

```diff
@@ -63,6 +59,20 @@ describe( 'DataController', () => {
 
 			expect( data.viewDocument ).to.be.instanceOf( ViewDocument );
 		} );
+
+		it( 'should create #htmlProcessor property', () => {
+			const stylesProcessor = new StylesProcessor();
+			const data = new DataController( model, stylesProcessor );
+
+			expect( data.htmlProcessor ).to.be.instanceOf( HtmlDataProcessor );
+		} );
+
+		it( 'should assign #htmlProcessor property to the #processor property', () => {
+			const stylesProcessor = new StylesProcessor();
+			const data = new DataController( model, stylesProcessor );
+
+			expect( data.htmlProcessor ).to.be.equal( data.processor );
```

Comment:
```suggestion
			expect( data.htmlProcessor ).to.equal( data.processor );
```

Just for consistency.